scripts/startup_xline.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
The filename `start_xline.sh` might be better.

Reply:
Done